docs/src/main/paradox/google-cloud-pub-sub-grpc.md

```diff
@@ -0,0 +1,139 @@
+
+Scala
+: @@snip (/google-cloud-pub-sub-grpc/src/test/scala/docs/scaladsl/ExampleUsage.scala) { #publish-single }
+
+Java
+: @@snip (/google-cloud-pub-sub-grpc/src/test/java/docs/javadsl/ExampleUsageJava.java) { #publish-single }
+
+
+Similarly to before, we can publish a batch of messages for greater efficiency.
+
+Scala
+: @@snip (/google-cloud-pub-sub-grpc/src/test/scala/docs/scaladsl/ExampleUsage.scala) { #publish-fast }
+
+Java
+: @@snip (/google-cloud-pub-sub-grpc/src/test/java/docs/javadsl/ExampleUsageJava.java) { #publish-fast }
+
+Finally, to automatically acknowledge messages and send messages to your own sink, once can do the following:
+
+Scala
+: @@snip (/google-cloud-pub-sub-grpc/src/test/scala/docs/scaladsl/ExampleUsage.scala) { #subscribe-auto-ack }
+
+Java
+: @@snip (/google-cloud-pub-sub-grpc/src/test/java/docs/javadsl/ExampleUsageJava.java) { #subscribe-auto-ack }
+
+## Subscribing
+
+To receive message from a subscription, first we create a `StreamingPullRequest` with a FQRS of a subscription and
+a deadline for acknowledgemts in seconds. Google requires that only the first `StreamingPullRequest` has the subscription
+and the deadline set. This connector takes care of that and clears up the subscription FQRS and the deadline for subsequent
+`StreamingPullRequest` messages.
+
+Scala
+: @@snip (/google-cloud-pub-sub-grpc/src/test/scala/docs/scaladsl/ExampleUsage.scala) { #subscribe }
+
+Java
+: @@snip (/google-cloud-pub-sub-grpc/src/test/java/docs/javadsl/ExampleUsageJava.java) { #subscribe }
+
+Here `pollInterval` is the time between `StreamingPullRequest` are sent when there are no messages in the subscription.
+
+Messages received from the subscription need to be acknowledged or they will be sent again. To do that create
+`AcknowledgeRequest` that contains `ackId`s of the messages to be acknowledged and send them to a sink
+created by `GooglePubSub.acknowledge`.
+
+Scala
+: @@snip (/google-cloud-pub-sub-grpc/src/test/scala/docs/scaladsl/ExampleUsage.scala) { #acknowledge }
+
+Java
+: @@snip (/google-cloud-pub-sub-grpc/src/test/java/docs/javadsl/ExampleUsageJava.java) { #acknowledge }
+
+
+## Running the test code
+
+@@@ note
+Integration test code requires Google Cloud Pub/Sub emulator running in the background. You can start it quickly using docker:
+
+`docker-compose up -d gcloud-pubsub-client`
+
+This will also run the Pub/Sub admin client that will create topics and subscriptions used by the
+integration tests.
+@@@
+
+Tests can be started from sbt by running:
+
+sbt
+:   ```bash
+    > google-cloud-pub-sub-grpc/test
+    ```
+
+There is also an @github[ExampleApp](/google-cloud-pub-sub-grpc/src/test/scala/docs/scaladsl/ExampleApp.scala) that can be used
+to test publishing to topics and receiving messages from subscriptions.
+
+To run the example app you will need to configure a project and Pub/Sub in Google Cloud and provide your own credentials.
+
+sbt
+:   &#9;
```

Comment:
Is that an HTML tab char?

Reply:
Yes it is. :) Workaround for https://github.com/lightbend/paradox/issues/161